cms/core/blocks/section_block.py

```diff
@@ -15,42 +18,39 @@
     RelatedLinksBlock,
 )
 
-if TYPE_CHECKING:
-    from wagtail.blocks import StructValue
-
 
 class SectionContentBlock(StreamBlock):
-    """The analysis section content StreamField block definition."""
+    """The core section content block definition."""
 
     rich_text = RichTextBlock()
     panel = PanelBlock()
+    embed = EmbedBlock(group="Media")
     image = ImageChooserBlock(group="Media")
     documents = DocumentsBlock(group="Media")
-    embed = EmbedBlock(group="Media")
+    related_links = RelatedLinksBlock(RelatedContentBlock(), icon="link")
     equation = MathBlock(group="DataVis", icon="decimal")
     ons_embed = ONSEmbedBlock(group="DataVis", label="ONS General Embed")
-    related_links = RelatedLinksBlock(RelatedContentBlock(), icon="link")
 
     class Meta:
         template = "templates/components/streamfield/stream_block.html"
-        block_counts: ClassVar[dict[str, dict]] = {"related_links": {"max_num": 1}}
 
 
 class SectionBlock(StructBlock):
-    """The analysis section block definition."""
+    """The core section block definition with headers."""
 
     title = HeadingBlock()
     content = SectionContentBlock()
 
+    class Meta:
+        block_counts: ClassVar[dict[str, dict]] = {"related_links": {"max_num": 1}}
+        template = "templates/components/streamfield/section_with_header_block.html"
+
     def to_table_of_contents_items(self, value: "StructValue") -> list[dict[str, str]]:
         """Convert the value to the table of contents component macro format."""
         return [{"url": "#" + slugify(value["title"]), "text": value["title"]}]
 
-    class Meta:
-        template = "templates/components/streamfield/analysis_section_block.html"
-
 
-class AnalysisStoryBlock(StreamBlock):
+class SectionStreamBlock(StreamBlock):
```

Comment:
This top level StreamBlock should live in `cms/core/blocks/stream_blocks.py`

IMHO it should be called `SectionStoryBlock` wherby `Story` indicates it is a top-level StreamBlock, as opposed to lower level ones (like `SectionContentBlock`)

Reply:
Yeah, I agree with the idea! 
I'm not sure if that's the best name for it tho, me and @sanjeevz3009 spent some time trying to figure out what it means. 
I'll try to come up with something more precise.

Reply:
I've moved and renamed it as advised in [74f5448](https://github.com/ONSdigital/dis-wagtail/pull/57/commits/74f5448c4a4689b595b6a5d4885cfc52db070094). Might change that if I come up with another name..